public/stylesheets/cards.css

```diff
@@ -154,6 +154,10 @@ fieldset {
 .field-input:active + label {
   background-color: rgba(1, 172, 1, 0.801);
 }
+.text-input {
```

Comment:
Line 61 identical styles:

```
.field-input {
  border-radius: 5px;
  padding: 3px;
}
```
Also since you renamed the class name for the inputs check if there's leftover code that's unused.

Reply:
I only renamed field inputs in `login` and `register` pages, as the styling I added to make inputs in `play-settings` and `play` pages look like buttons made the inputs in `login` and `register` be hidden.